pages/product_list.html

```diff
@@ -0,0 +1,294 @@
+                            </div>   
+                    </div>
+
+
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/19.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Abry Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
+                            <p class="card-discount">60% off</p>
+                            </div>
+                            <div class="card-footer ">
+                                <button class="btn btn-primary "> <i class="fas  fa-shopping-cart"></i> Add to Cart </button>
+                                <button class="btn btn-primary "> Buy Now </button>
+                            </div>
+                            </div>   
+                    </div>
+
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/28.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Woman Pink Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
+                            <p class="card-discount">60% off</p>
+                            </div>
+                            <div class="card-footer ">
+                                <button class="btn btn-primary "> <i class="fas  fa-shopping-cart"></i> Add to Cart </button>
+                                <button class="btn btn-primary "> Buy Now </button>
+                            </div>
+                            </div>   
+                    </div>
+
+
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/29.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Soho Bry Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
+                            <p class="card-discount">60% off</p>
```

Comment:
Discount value is not correct. Not that it matters but change it if you can.

Reply:
it is correct